source/index.html.md

```diff
@@ -851,14 +854,14 @@ print(result.json())
 ```shell
 curl "https://api.feathery.io/api/form/submission/" \
     -X POST \
-    -d "{'fields': {'age': 21}, 'user_id': 'alice@feathery.io', 'forms': ['My Form'], 'complete': True}" \
+    -d "{'fields': {'NumberField1': 21, 'ef5ed054-73de-4463-ba61-82c36aca5afc': 'Test Value'}, 'user_id': 'alice@feathery.io', 'forms': ['My Form'], 'complete': True}" \
```

Comment:
let's remove these from the code examples, we shouldn't necessarily encourage the use